### DISCUSSION TITLE
Official Material X support

### DISCUSSION THREAD

**Discussion by JeffGillin** [bevyengine/bevy#14938] [Ideas]
As of 4.2, Blender now has ability to export MaterialX shaders which is huge.

Looks like bevy support has been started. I just wanted to suggest an official port inside bevy.

https://github.com/killercup/materialx

https://projects.blender.org/blender/blender/pulls/122575

**Comment by killercup** [comment]
Hi, thanks opening this! I'm the one who started the bevy plugin! 👋 

Right now, I'm a bit busy with other work but I've made some progress before my summer break, and I plan on picking it up again soon. (Sooner if anyone can sponsor this work, otherwise it's based on how much free time I can sacrifice.)

I added a bunch of notes to the repo, too, incl. [the original plan](https://github.com/killercup/materialx/blob/1a26676d37df98c7f2bce28f8e06176fc51ffff8/notes/plan.md) as discussed on discord. If anyone wants to chat with me about this/collaborate, ping me here or on discord :)
